gitevanhsu/book-reviews
# Feature/leave book rating/reviews

**Description**
1. User can't leave rating or review before sign in.
2. Member can leave books rating or review.
3. Member can edit rating and review.
4. Member can delete rating and review.
5. Show all rating and review immediately.

**Test Plan**
1. If user didn't sign in can't leave rating or review.
2. Try leave rating and review.
3. Edit rating or review.
4. Delete Rating and Review.
5. Check Firestore's data change as expect.
